crates/ursa-gateway/src/server/route/api/v1/get.rs

```diff
@@ -96,7 +157,13 @@ pub async fn get_block_handler(
     // 1. maintain N workers keep track of indexing data
     // 2. cherry-pick closest node
     // 3. cache TTL
-    (StatusCode::OK, Json(json!(indexer_response)))
+    (
```

Comment:
I am sure y’all are already thinking about it but we should be steaming back data.